config/rbac/appplatform_role_binding.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: supervisor-properties-reader-binding
```

Comment:
I'm leaning towards calling the role and rolebinding files. 

config/rbac/supervisor_properties_role.yaml and 
config/rbac/supervisor_properties_role_binding.yaml

Reply:
Fixed.